logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/ObserveConversationViewUseCaseImpl.kt

```diff
@@ -0,0 +1,17 @@
+package com.wire.kalium.logic.feature.conversation
+
+import com.wire.kalium.logic.data.conversation.ConversationDetails
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import kotlinx.coroutines.flow.Flow
+
+fun interface ObserveConversationViewUseCase {
+    suspend operator fun invoke(): Flow<List<ConversationDetails>>
+}
+
+internal class ObserveConversationViewUseCaseImpl(
```

Comment:
I prefer to keep the old name ObserveConversationListDetailsUseCase. I don't think we should expose the "View" name at all.

Reply:
Yes, I missed that! thanks